### ISSUE TITLE
Package organisation

### ISSUE THREAD

**Issue by LeventeNagy** [totaljs/framework#336]
Hi @petersirka 

I read the following in the new feature list

<pre>
Packages can be loaded in framework structure (/controllers/, /modules/) exports.booting = true
</pre>

I took a look at the docs but was not able to find any info.

Could you please give an example how to structure packages in framework structure way?

Thank you

**Comment by petersirka** [comment]
Hi @LeventeNagy,
ohh shit yes, you're right. I forgot on documentation.

If your package contains:

- `your.package/controllers/some-controller-name.js`
- `your.package/models/some-model-name.js`
- `your.package/index.js` --> `exports.booting = true`

Then the framework unpacks all files and evaluates them as classic framework files with except static files and views.

So the files will be visible for the framework like this:

- `/controllers/some-controller-name.js`
- `/models/some-model-name.js`

In short: package can contains a part of application.
Do you undestand?

**Comment by LeventeNagy** [comment]
Ah great. Thank you now its fine
